doc/api/stream.md

```diff
@@ -1288,8 +1288,10 @@ changes:
 * `encoding` {string} Encoding of string chunks. Must be a valid
   `Buffer` encoding, such as `'utf8'` or `'ascii'`.
 
-Passing `chunk` as `null` signals the end of the stream (EOF), after which no
-more data can be written.
+Passing `chunk` as `null` signals the end of the stream (EOF) and behaves the
+same as `readable.push(null)`, after which no more data can be written. Note
+that the EOF signal is put at the end of the buffer and any buffered data will
+still be flushed.
```

Comment:
Micro nit: Remove `Note that` from the last sentence?

Reply:
I'm surprised the [linter](https://travis-ci.com/nodejs/node/jobs/246209882) didn't catch this.